static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -0,0 +1,145 @@
+import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
+
+import Input from 'sentry/components/forms/controls/input';
+import Slider from 'sentry/components/forms/controls/rangeSlider/slider';
+import SliderAndInputWrapper from 'sentry/components/forms/controls/rangeSlider/sliderAndInputWrapper';
+import SliderLabel from 'sentry/components/forms/controls/rangeSlider/sliderLabel';
+import type SliderProps from 'sentry/components/forms/controls/rangeSlider/sliderProps';
+import {t} from 'sentry/locale';
+import {defined} from 'sentry/utils';
+
+function RangeSlider({
+  value,
+  allowedValues,
+  showCustomInput,
+  name,
+  disabled,
+  placeholder,
+  formatLabel,
+  className,
+  onBlur,
+  onChange,
+  forwardRef,
+  showLabel = true,
+  ...props
+}: SliderProps) {
+  const [sliderValue, setSliderValue] = useState(
+    allowedValues ? allowedValues.indexOf(Number(value || 0)) : value
+  );
+
+  useEffect(() => {
+    updateSliderValue();
+  }, [value]);
+
+  function updateSliderValue() {
```

Comment:
Nit, but I would probably try and define fns before their use effect calls. This one is small enough so that you could inline it into the entire useEffect.

Reply:
This is existing code unrelated to the exports. Moving to new PR https://github.com/getsentry/sentry/pull/33219